crypto/verifier.go

```diff
@@ -31,8 +31,13 @@ import (
 	"github.com/google/trillian/crypto/sigpb"
 )
 
-// ErrVerify occurs whenever signature verification fails.
-var ErrVerify = errors.New("signature verification failed")
+var (
+	errVerify = errors.New("signature verification failed")
```

Comment:
Ok, from https://blog.golang.org/error-handling-and-go, I got the impression that type assertions are the way to go, but I'll bear this in mind for the future. However, in this case, ErrVerify is not always returned when signature verification fails, as only the verifyECDSA func returns it (verifyRSA doesn't).